<ui version="4.0" >
 <class>TasterMainWindow</class>
 <widget class="QMainWindow" name="TasterMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>63</y>
     <width>700</width>
     <height>470</height>
    </rect>
   </property>
   <widget class="QDockWidget" name="dwJsInput" >
    <property name="windowModality" >
     <enum>Qt::NonModal</enum>
    </property>
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>80</y>
      <width>380</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="floating" >
     <bool>true</bool>
    </property>
    <property name="allowedAreas" >
     <set>Qt::AllDockWidgetAreas</set>
    </property>
    <property name="windowTitle" >
     <string>This is JavaScript Input</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>23</y>
       <width>380</width>
       <height>98</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QListWidget" name="teJsInput" />
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLineEdit" name="lineEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="pushButton" >
          <property name="text" >
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dwPreview" >
    <property name="windowModality" >
     <enum>Qt::NonModal</enum>
    </property>
    <property name="geometry" >
     <rect>
      <x>9</x>
      <y>58</y>
      <width>258</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="floating" >
     <bool>true</bool>
    </property>
    <property name="features" >
     <set>QDockWidget::AllDockWidgetFeatures</set>
    </property>
    <property name="allowedAreas" >
     <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
    </property>
    <property name="windowTitle" >
     <string>This is a Preview</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>23</y>
       <width>258</width>
       <height>338</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QWebView" name="wkPreview" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>240</width>
          <height>320</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>240</width>
          <height>320</height>
         </size>
        </property>
        <property name="url" >
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="dwLogs" >
    <property name="windowModality" >
     <enum>Qt::NonModal</enum>
    </property>
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>240</y>
      <width>362</width>
      <height>209</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="floating" >
     <bool>true</bool>
    </property>
    <property name="allowedAreas" >
     <set>Qt::AllDockWidgetAreas</set>
    </property>
    <property name="windowTitle" >
     <string>This is Alert Log</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>23</y>
       <width>362</width>
       <height>186</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QListWidget" name="lwAlertLog" />
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actOpen" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actReload" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTools" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>25</y>
     <width>652</width>
     <height>38</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="allowedAreas" >
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>true</bool>
   </attribute>
   <addaction name="actReload" />
   <addaction name="actOpen" />
  </widget>
  <widget class="QToolBar" name="toolBar2" >
   <property name="geometry" >
    <rect>
     <x>652</x>
     <y>25</y>
     <width>48</width>
     <height>38</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actResize" />
  </widget>
  <widget class="QStatusBar" name="statusBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>533</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actReload" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/icons/reload.svg</normaloff>:/icons/reload.svg</iconset>
   </property>
   <property name="text" >
    <string>Reload</string>
   </property>
  </action>
  <action name="actOpen" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/icons/fileopen.svg</normaloff>:/icons/fileopen.svg</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actResize" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/icons/resize.png</normaloff>:/icons/resize.png</iconset>
   </property>
   <property name="text" >
    <string>Resize</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actReload</sender>
   <signal>triggered()</signal>
   <receiver>wkPreview</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
